bindata/etcd/restore-pod.yaml

```diff
@@ -48,15 +48,18 @@ spec:
         fi
 
         UUID=$(uuidgen)
+        UUID=${UUID:0:10}
```

Comment:
just use the whole ID or another unique string

Reply:
done